apm-agent-core/src/main/java/co/elastic/apm/agent/report/serialize/DslJsonSerializer.java

```diff
@@ -106,6 +109,18 @@
     @Nullable
     private OutputStream os;
 
+    static {
+        Method getOurStackTrace;
+        try {
+            getOurStackTrace = Throwable.class.getDeclaredMethod("getOurStackTrace");
+            getOurStackTrace.setAccessible(true);
```

Comment:
I think this comes with a warning it is not going to be allowed in future versions, maybe since 9.